lib/src/screens/exchange/exchange_page.dart

```diff
@@ -100,271 +100,197 @@ class ExchangePage extends BasePage {
+                                        },
+                                        onPushAddressBookButton: (context) async {
+                                          final domain = exchangeViewModel.receiveAddress;
+                                          final ticker = exchangeViewModel.receiveCurrency.title.toLowerCase();
+                                          exchangeViewModel.receiveAddress =
+                                              await fetchParsedAddress(context, domain, ticker);
+                                        },
+                                      )),
+                            )
+                          ],
+                        ),
                       ),
-                    ),
-                    Padding(
-                      padding: EdgeInsets.only(top: 12, left: 24),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        children: [
-                          StandardCheckbox(
-                            key: checkBoxKey,
-                            value: exchangeViewModel.isFixedRateMode,
-                            caption: S.of(context).fixed_rate,
-                            onChanged: (value) =>
-                            exchangeViewModel.isFixedRateMode = value,
-                          ),
-                        ],
-                      )
-                    ),
-                    SizedBox(height: 30),
-                    _buildTemplateSection(context)
+                      Padding(
+                          padding: EdgeInsets.only(top: 12, left: 24),
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.start,
+                            children: [
+                              StandardCheckbox(
+                                key: checkBoxKey,
+                                value: exchangeViewModel.isFixedRateMode,
+                                caption: S.of(context).fixed_rate,
+                                onChanged: (value) => exchangeViewModel.isFixedRateMode = value,
+                              ),
+                            ],
+                          )),
+                      SizedBox(height: 30),
+                      _buildTemplateSection(context)
                     ],
                   ),
                 ),
-                bottomSectionPadding:
-                    EdgeInsets.only(left: 24, right: 24, bottom: 24),
+                bottomSectionPadding: EdgeInsets.only(left: 24, right: 24, bottom: 24),
                 bottomSection: Column(children: <Widget>[
                   Padding(
                     padding: EdgeInsets.only(bottom: 15),
                     child: Observer(builder: (_) {
                       final description = exchangeViewModel.isFixedRateMode
+                          ? exchangeViewModel.isAvailableInSelected
```

Comment:
Sorry for the formatting, but the file was too un-organized to modify, anyway, here is the change that I added

Reply:
here is one